e2e-tests/playwright/e2e/plugins/github-insights/github-insights.spec.ts

```diff
@@ -0,0 +1,93 @@
+import { expect, test } from "@playwright/test";
+import { UIhelper } from "../../../utils/ui-helper";
+import { Common } from "../../../utils/common";
+import { UI_HELPER_ELEMENTS } from "../../../support/pageObjects/global-obj";
+
+// Test to verify if the GitHub Insights plugin is installed - runs independently
+test("Verify GitHub Insights plugin is installed", async ({ page }) => {
+  const common = new Common(page);
+  await common.loginAsGuest();
+
+  const uiHelper = new UIhelper(page);
+
+  // Navigate to the Administration in the sidebar
```

Comment:
Excessive Comments:
While comments are useful, some explain actions that are already obvious from the function names.